v1.12/openshift/PRODUCT.yaml

```diff
@@ -0,0 +1,6 @@
+vendor: Red Hat
+name: OpenShift Online and OpenShift Container Platform by Red Hat
+version: v4.0.0
+website_url: https://www.openshift.com
+documentation_url: https://docs.openshift.com
+product_logo_url: https://assets.openshift.net/content/subdomain/openshift-logo-horizontal.svg
```

Comment:
From https://www.openshift.com/about/logos it looks like you've switched OpenShift logos to include the Red Hat name.

The one we have at https://landscape.cncf.io/logos/red-hat-open-shift.svg looks correct.

Reply:
Updated to 
https://cdn2.hubspot.net/hubfs/4305976/images/openshift-legacy/logos/openshift/Logotype_RH_OpenShift_wLogo_RGB_Black.svg which is the current brand guideline.